protostar/starknet/cheaters/contracts.py

```diff
@@ -288,3 +288,9 @@ async def call(
             general_config=StarknetGeneralConfig(),
         )
         return result.retdata
+
+    def store(self, name: str, value: Any) -> None:
+        self.cheatable_state.cairo_storage[name] = value
+
+    def load(self, name: str) -> Any:
+        return self.cheatable_state.cairo_storage.get(name)
```

Comment:
TODO: Think about the type here. We could basically store anything but we can narrow it down to `CairoOrPythonData`, `str`, `int` or something